### ISSUE TITLE
Support sending elasticsearch cluster health metrics

### ISSUE THREAD

**Issue by allenj** [influxdata/telegraf#310]
Currently only the /_nodes/stats and /_nodes/_local/stats are hit. Cluster health would also be beneficial (https://www.elastic.co/guide/en/elasticsearch/guide/current/_cluster_health.html).

**Comment by sparrc** [comment]
sounds good, would love to get a PR for this from anyone familiar with Elastic Search :+1:

**Comment by jipperinbham** [comment]
@allenj are you just thinking the top level health call or also including `?level=indices`?

**Comment by allenj** [comment]
I was thinking ```?level=indices``` if possible so you know the specific index that is having issues.

**Comment by jipperinbham** [comment]
Ok, I'll see how hard it will be to add `?level=indices` in addition to general cluster health.

**Comment by otoolep** [comment]
I think `indices` should be tags

**Comment by jipperinbham** [comment]
@otoolep sounds good, so for the following input
```JSON
"indices": {
      "v1": {
         "status": "green",
         "number_of_shards": 10,
         "number_of_replicas": 1,
         "active_primary_shards": 10,
         "active_shards": 20,
         "relocating_shards": 0,
         "initializing_shards": 0,
         "unassigned_shards": 0
      },
      "v2": {
         "status": "red", 
         "number_of_shards": 10,
         "number_of_replicas": 1,
         "active_primary_shards": 0,
         "active_shards": 0,
         "relocating_shards": 0,
         "initializing_shards": 0,
         "unassigned_shards": 20 
      }
}
```

we'd process the tags as (semi-pseudo code)

```Go
for k, v := range n.Indices {
  for attr, value := range v.(map[string]interface{}) {
    tags["index_"+k+"_"+attr] = value
  }
}
```

**Comment by otoolep** [comment]
Thanks @jipperinbham -- do you need to prefix the value names?

If I was to pump that data into InfluxDB, I would do this (in terms of line protocol):

`indices,index=v1 status=green,number_of_shards=10,.....`

At least I think so. This schema would allow me to know how many relocating shards there were, across the cluster and across time, with a query like so:

`SELECT relocating_shards FROM indices`

or if I just wanted the counts for a single index:

`SELECT relocating_shards FROM indices WHERE index='v2'`

Or the number of shards, group by index:

`SELECT number_of_shards FROM indices GROUP BY index`

You the idea.

**Comment by jipperinbham** [comment]
Ah, yes, I see the data model now.

**Comment by sparrc** [comment]
closed by #327
